src/model/NPMResolver.ts

```diff
@@ -74,9 +74,31 @@ export class NPMResolver {
         }
         console.log(cgreen`✓`, `[${pluginName}] (NPM) dependencies successfully installed`);
         NPMResolver.createPluginHashFile(assetsPath);
+        NPMResolver.removePluginSymlinks(pluginName, assetsPath);
         return true;
     }
 
+    /**
+     * Removes symlinks inside plugin node_modules folder
+     * @param pluginName Plugin name
+     * @param assetsPath Assets folder path
+     * @private
+     */
+    private static removePluginSymlinks(pluginName: string, assetsPath: string): void {
+        const isWindows = process.platform === 'win32';
+        if (isWindows) {
```

Comment:
Are the symlinks only created on Windows?
In addition to the failing build, which may be a problem only on Windows,
I noticed that IntelliJ picks up the second Git repository root. It complains after the symlink is removed, because it no longer finds that repository root. That could be a problem on other systems as well.

Reply:
Symlinks are currently reported only on Windows. If problems appears on others OSs, we can easily fix it 😉